geode-core/src/main/java/org/apache/geode/internal/cache/AbstractRegionMap.java

```diff
@@ -1118,13 +1119,8 @@ public void txApplyDestroy(Object key, TransactionId txId, TXRmtEvent txEvent,
                     clearOccured /* Clear Conflciting with the operation */);
                 if (invokeCallbacks) {
                   switchEventOwnerAndOriginRemote(cbEvent, hasRemoteOrigin);
-                  if (pendingCallbacks == null) {
-                    owner.invokeTXCallbacks(EnumListenerEvent.AFTER_DESTROY, cbEvent,
-                        true/* callDispatchListenerEvent */);
-                  } else {
-                    pendingCallbacks.add(cbEvent);
-                    cbEventInPending = true;
-                  }
+                  pendingCallbacks.add(cbEvent);
+                  cbEventInPending = true;
```

Comment:
What does the prefix "cb" mean? I know this is a pre-existing variable name but we should try to remove all abbreviations and otherwise change variable names to be more expressive and correct.

Reply:
I have changed "cb" to "callback" and updated the pull request